Topic3/src/org/maugiam/tdd/test/AmountConverterTest.java

```diff
@@ -0,0 +1,35 @@
+package org.maugiam.tdd.test;
+
+import junit.framework.TestCase;
+
+import org.junit.Test;
+import org.maugiam.tdd.main.AmountConverter;
+
+public class AmountConverterTest extends TestCase {
+
+	AmountConverter converter;
+
+	public void setUp() {
+		converter = new AmountConverter();
+	}
+
+	@Test
+	public void testCreateConverter() {
+		assertNotNull(converter);
+	}
+
+	@Test
+	public void testSimpleAmountConver() {
+		assertEquals("One dollar", converter.convert(1.00));
+	}
+
+	@Test
+	public void testComplexAmountConver() {
+		assertEquals("Two thousand five hundred twenty-three and 04/100 dollars", converter.convert(2523.04));
+		System.out.println(converter.convert(1217.0));

```

Comment:
Yes, it was created first but it wasn't working so i wanted to know what was happening, and the sysout was helpful for that.